src/app/functions/server/getUserBreaches.ts

```diff
@@ -95,10 +95,16 @@ export async function getUserBreaches({
  * NOTE: new function to replace getUserBreaches
  *
  * @param user
+ * @param user.user
+ * @param user.countryCode
  */
-export async function getSubscriberBreaches(
-  user: Session["user"],
-): Promise<SubscriberBreach[]> {
+export async function getSubscriberBreaches({
```

Comment:
is there an advantage of passing in an object of `{user, countryCode}` vs. two params `(user, countryCode)`?

Reply:
TS makes it easier to know what arguments to pass to a function. Generally, this is a good way to know which one a function takes. We do it in a couple of places, but I don’t think we follow a consistent pattern.